src/api/public/apidocs-new/paths/source_project_name_config.yaml

```diff
@@ -0,0 +1,56 @@
+get:
+  summary: Get project's configurations
+  description: |
+    Read configurations for the specified project
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - name: rev
+      in: query
+      schema:
+        type: string
+      required: true
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        text/plain:
+          schema:
+            type: string
+          example: |
+            # temporary should be fixed in SUSE:SLE-15-SP4:GA
+            Prefer: -libudev-devel
+            Prefer: -libudev-devel-32bit
+            ...
+  tags:
+    - Sources - Projects
+
+put:
+  summary: Update project's configurations
+  description: |
+    Update configurations for the specified project
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - name: comment
+      in: query
+      schema:
+        type: string
+  requestBody:
+    description: configurations you want to update
+    content:
+      text/plain:
+        schema:
+          type: string
+        example: |
+          # temporary should be fixed in SUSE:SLE-15-SP4:GA
+          Prefer: -libudev-devel
+          Prefer: -libudev-devel-32bit
+          ...
+  responses:
```

Comment:
It's already known. The system only responds when everything is OK. Otherwise, it raises a backend exception resulting 406 HTTP for all other errors.

Reply:
If I force to raise any of the exceptions in [this line](https://github.com/openSUSE/open-build-service/blob/e05b4437d9c73f7ddba828bcf8eec2b999b41a95/src/api/app/controllers/source_project_config_controller.rb#L62), I get a 404 response. Souldn't they be documented?